code/lib/cli-storybook/package.json

```diff
@@ -19,15 +19,67 @@
     "url": "https://opencollective.com/storybook"
   },
   "license": "MIT",
-  "bin": {
-    "sb": "./index.js",
-    "storybook": "./index.js"
+  "author": "Storybook Team",
+  "type": "module",
+  "exports": {
+    "./bin/index.cjs": {
+      "node": "./bin/index.cjs",
+      "require": "./bin/index.cjs"
+    },
+    "./package.json": "./package.json"
+  },
+  "bin": "./bin/index.cjs",
+  "files": [
+    "bin/**/*",
+    "dist/**/*",
+    "README.md",
+    "!src/**/*"
+  ],
+  "scripts": {
+    "check": "node --loader ../../../scripts/node_modules/esbuild-register/loader.js -r ../../../scripts/node_modules/esbuild-register/register.js ../../../scripts/prepare/check.ts",
+    "prep": "node --loader ../../../scripts/node_modules/esbuild-register/loader.js -r ../../../scripts/node_modules/esbuild-register/register.js ../../../scripts/prepare/bundle.ts"
   },
   "dependencies": {
-    "storybook": "workspace:*"
+    "@babel/core": "^7.24.4",
+    "@babel/types": "^7.24.0",
+    "@storybook/codemod": "workspace:*",
+    "@types/semver": "^7.3.4",
+    "chalk": "^4.1.0",
+    "commander": "^6.2.1",
+    "create-storybook": "workspace:*",
```

Comment:
This depends on `create-storybook` for the sandbox command

Reply:
If this is necessary purely for the sandbox command, shouldn't we move it to the `devDependencies` instead?

Reply:
Good point, I tried to, but I couldn't get it pre-bundled, because of some deps of `create-storybook` that are not able to be pre-bundled.

Reply:
The sandbox command might be only used by us, but it's still public.

We can't make this a devDependency AFAICS